product_docs/docs/pgd/5/cli/index.mdx

```diff
@@ -13,15 +13,15 @@ directoryDefaults:
   description: "The PGD Command Line Interface (CLI) is a tool to manage your EDB Postgres Distributed cluster"
 ---
 
-The EDB Postgres Distributed Command Line Interface (PGD CLI) is a tool for managing your EDB Postgres Distributed cluster. It allows you to run commands against EDB Postgres Distributed clusters. It may be installed automatically on systems within a TPA-deployed PGD cluster or it can be installed manually on systems that can connect to any PGD cluster, including BigAnimal Distributed High Availability PGD clusters or PGD clusters deployed using the EDB PGD for Kubernetes operator.
+The EDB Postgres Distributed Command Line Interface (PGD CLI) is a tool for managing your EDB Postgres Distributed cluster. It allows you to run commands against EDB Postgres Distributed clusters. It can be installed automatically on systems in a TPA-deployed PGD cluster. Or it can be installed manually on systems that can connect to any PGD cluster, including BigAnimal distributed high-availability PGD clusters or PGD clusters deployed using the EDB PGD for Kubernetes operator.
 
-See [Installing PGD CLI](installing_cli) for information about how to install PGD CLI, both automatically with Trusted Postgres Architect and manually.
+See [Installing PGD CLI](installing_cli) for information about how to install PGD CLI automatically with Trusted Postgres Architect and manually.
```

Comment:
I don't think it tells them how to install with TPA (just says again that it will be done) = the installing chapter is really about manually installing depending on how the PGD cluster was deployed